esx/PXELinux.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: PXELinux
+name: ESXi PXELinux
+oses:
+- ESXi 5
+- ESXi 5.5
+- ESXi 6
+%>
+
+DEFAULT linux
+LABEL linux
+KERNEL undionly.kpxe
+APPEND dhcp && chain <%= foreman_url('iPXE') %>
```

Comment:
At this point I get `Loading undionly.kpxe ... OK. Booting kernel failed: Invalid argument.`

This is what that chain resolves to:

```
#!ipxe
kernel -n mboot.c32 http://localhost:9000//mboot.c32
imgargs mboot.c32 -c http://localhost:9000// text ks=http://toshiba.lobatolan.home:3000/unattended/provision?token=31aae981-d3aa-473d-b394-41242613cd95; ignoreHeadless="True"
boot mboot.c32
```

Any ideas? From what I see in the installation media there's no attempt to even GET mboot.c32 so I guess that's not a problem